.gitignore

```diff
@@ -25,6 +25,9 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
+<<<<<<< HEAD
+.vscode
+=======
```

Comment:
looks like a merge conflict

Reply:
i added and changed that so the vs code settings would stay out of the repo.... gonna squash and merge it in